lib/strings.js

```diff
@@ -408,7 +408,8 @@ const Strings = new LocalizedStrings({
       noGoBack: 'No, regresar',
     },
     logout: {
-      popupText: '¿Estás segura de que quieres cerrar sesión?',
+      popupText:
+        '¿Está usted seguro de que desea eliminar TODOS sus datos guardados?',
```

Comment:
@markkhoo I don't understand this change- logging out will not trigger a user account delete, so why are we copying that text here?

Reply:
I think I replaced the wrong string here. Good catch!